### PR TITLE
Social share

### PR DESCRIPTION
**Pull request by amit-webkul**

### CODE REVIEW

File: packages/Webkul/Shop/src/Http/Controllers/API/CompareController.php [hunk 1]
```diff
@@ -91,6 +91,7 @@ public function destroy(): JsonResource
 
         if (! $success) {
             return new JsonResource([
+                'data'     => CompareResource::collection($this->compareItemRepository->get()),
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/API/CompareController.php:94]: Change this and add customer_id.

File: packages/Webkul/SocialShare/src/Providers/EventServiceProvider.php [hunk 2]
```diff
@@ -14,7 +15,7 @@ class EventServiceProvider extends ServiceProvider
     public function boot()
     {
         Event::listen('bagisto.shop.products.view.description.before', function ($viewRenderEventManager) {
-            $viewRenderEventManager->addTemplate('social_share::share');
+            $viewRenderEventManager->addTemplate('socialshare::share');
```
Comment (devansh-webkul) [inline packages/Webkul/SocialShare/src/Providers/EventServiceProvider.php:18]: snake case

File: packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php [hunk 3]
```diff
@@ -7,13 +8,13 @@ class SocialShareServiceProvider extends ServiceProvider
 {
     public function boot()
     {
-        $this->loadViewsFrom(__DIR__ . '/../Resources/views', 'social_share');
+        $this->loadViewsFrom(__DIR__ . '/../Resources/views/', 'socialshare');
```
Comment (devansh-webkul) [inline packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php:11]: snake case

File: packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php [hunk 4]
```diff
@@ -7,13 +8,13 @@ class SocialShareServiceProvider extends ServiceProvider
...
 
-        $this->loadJSONTranslationsFrom(__DIR__ . '/../Resources/lang');
+        $this->loadTranslationsFrom(__DIR__ . '/../Resources/lang', 'socialshare');
```
Comment (devansh-webkul) [inline packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php:13]: snake case

File: packages/Webkul/SocialShare/src/Providers/EventServiceProvider.php (hunk 2 above)
Comment (amit-webkul) [reply packages/Webkul/SocialShare/src/Providers/EventServiceProvider.php]: fixed

File: packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php (hunk 3 above)
Comment (amit-webkul) [reply packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php]: fixed

File: packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php (hunk 4 above)
Comment (amit-webkul) [reply packages/Webkul/SocialShare/src/Providers/SocialShareServiceProvider.php]: fixed

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**